application/src/main/resources/extensions/attachment-local-policy.yaml

```
@@ -35,6 +35,29 @@ spec:
           name: location
           label: 存储位置
           help: ~/.halo2/attachments/upload 下的子目录
+        - $formkit: text
+          name: maxFileSize
+          label: 最大单文件大小
+          help: "0 表示不限制，示例: 5KB, 10MB, 1GB"
+        - $formkit: select
+          name: allowedFileTypes
+          label: 文件类型限制
+          help: 限制允许上传的文件类型
+          options:
+            - label: 无限制
+              value: ALL
+            - label: 图片
+              value: IMAGE
+            - label: SVG
+              value: SVG
+            - label: 视频
+              value: VIDEO
+            - label: 音频
+              value: AUDIO
+            - label: 文档
+              value: DOCUMENT
+            - label: 压缩包
+              value: ARCHIVE
```

这里可以使用 `multiple` 来支持多选。

```suggestion
        - $formkit: select
          name: allowedFileTypes
          label: 文件类型限制
          help: 限制允许上传的文件类型
          multiple
          options:
            - label: 无限制
              value: ALL
            - label: 图片
              value: IMAGE
            - label: SVG
              value: SVG
            - label: 视频
              value: VIDEO
            - label: 音频
              value: AUDIO
            - label: 文档
              value: DOCUMENT
            - label: 压缩包
              value: ARCHIVE
```